docs/api-process/check-out.md

```diff
@@ -3,4 +3,108 @@ id: check-out
 title: How to Create Checkout?
 ---
 
-This topic describes the process of check out.
\ No newline at end of file
+Below is a description of a checkout process. We assume that at this stage you have already completed the steps included in the Getting Started section of this chapter and you are familiar with basic setup of Saleor GraphQL API.
+
+The below process describes the key milestones in the checkout process flow in Saleor. There are also additional steps that may occur along the way. However, the purpose of this instruction is to deliver a base reference for the user to work with.    
+
+The code snippets included in this section may be run in ![Playground](api/playground.md) or your preferred HTTP client.
+
+### Empty checkout
+
+The checkout process in Saleor may run through several scenarios depending on such factors like, for example, if the user is logged in or not, at the moment of adding items to the cart. 
+
+Saleor does not create ‘empty checkouts’ when a not-logged-in user is adding items to the cart. In such instances, the current cart is only saved on the browser level. It is not saved anywhere on the server.
+
+The actual checkout process is started after the user clicks _Checkout_ in their cart. If the user is already logged in - the process is commenced. If not, the user is prompted to log into their account first.
```

Comment:
I think, it is worth mentioning that the login process is optional. The user can continue as an anonymous user.